lte/protos/session_manager.proto

```diff
@@ -27,6 +27,7 @@ message RuleRecord {
   uint64 bytes_tx = 3;
   uint64 bytes_rx = 4;
   string ue_ipv4 = 5;
+  string ue_ipv6 = 6;
```

Comment:
@uri200, @themarwhal 
Is it okay if I use string for ipv6 addr here?

Reply:
sure, we store ips on strings but then for consistency, can we change the format from bytes to strings too on ipv6_addr for the fields below?

```
message ActivateFlowsRequest {
  SubscriberID sid = 1;
  string ip_addr = 2; // Subscriber session ipv4 address
  repeated string rule_ids = 3; // List of static rules obtained from PCRF
  repeated PolicyRule dynamic_rules = 4;  // List of dynamic rules obtained from PCRF
  RequestOriginType request_origin = 5; // Activate flow source (GX/GY)
  magma.lte.AggregatedMaximumBitrate apn_ambr = 6; // APN AMBR info, passed at the time of session creation
  bytes ipv6_addr = 7; // Subscriber session ipv6 address
}
```

```
message DeactivateFlowsRequest {
  SubscriberID sid = 1;
  repeated string rule_ids = 2;
  RequestOriginType request_origin = 3;
  string ip_addr = 4; // Subscriber session ipv4 address
  bytes ipv6_addr = 5; // Subscriber session ipv6 address
}
```

Reply:
Lets keep it as is for now to be consistent with ipv6 being bytes in flow match